Runtime/Tracking/Velocity/RigidbodyVelocityTracker.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Zinnia.Tracking.Velocity
+{
+    using Malimbe.MemberClearanceMethod;
+    using Malimbe.PropertySerializationAttribute;
+    using Malimbe.XmlDocumentationAttribute;
+    using UnityEngine;
+
+    /// <summary>
+    /// Retrieves the velocity for a <see cref="Rigidbody"/>.
+    /// </summary>
+    public class RigidbodyVelocityTracker : VelocityTracker
+    {
+        /// <summary>
+        /// The source to track and estimate velocities for.
+        /// </summary>
+        [Serialized, Cleared]
+        [field: DocumentedByXml]
+        public Rigidbody Source { get; set; }
+
+        /// <summary>
+        /// The velocity to return.
+        /// </summary>
+        [Serialized]
+        [field: DocumentedByXml]
+        public Vector3 Velocity { get; set; }
```

Comment:
why can you specify a velocity (and angular velocity)? these don't seem to be used anywhere, only Source is used.

Reply:
Ah i forgot to remove those properties after dup from constant velocity tracker
It should only have the Source property

Reply:
removed